### DISCUSSION TITLE
Failed to load index file: DISK2CPU : Failed to load deleted docs from

### DISCUSSION THREAD

**Discussion by dubeamit** [milvus-io/milvus#7274] [Q&A and General discussion]
I have two servers in cluster mode one in RW mode & the other in RO mode, the milvus version is v0.10.2.
I'm trying to upgrade the milvus from v0.10.2 to v1.1.1. 
So in a new servers(two server same as above in cluster mode) I installed the milvus v1.1.1 migrated the **db** folder the other server. But when I try to perfom a search on it I get the following error

`[2021-08-24 21:12:23,010][ERROR][SERVER][OnExecute][reqsched_thread] [search][0] Query fail: Failed to load index file: DISK2CPU : Failed to load deleted docs from /var/lib/milvus/db/tables/collection_name_here/1625057705479894000/1625057705479894000`

But on the new server with milvus v0.10.2 the search works.
I had an old **db** folder from milvus v0.10.2 which I used on the new milvus v1.1.1, here I could perform the search without any issues.

**Comment by yhmo** [comment]
#7275
